### DISCUSSION TITLE
if and why mpp_get_boundary should be called when .not. bounded_domain

### DISCUSSION THREAD

**Discussion by TingLei-NOAA** [ufs-community/ufs-weather-model#1020] [Q&A]
Dear UFS developers, 
Would you tell me if and why the  mpp_get_boundary should **only** be called when  bounded_domain is not true (namely, global runs)? 
From the codes itself, seems this should work for regional runs (bounded_domain=.true.).  But, it was found there would be problems when it was used in jedi 's D-A wind conversion for fv3-lam (regional fv3).  And in UFS  dyn_core.F90, there is ` if( it==n_split .and. gridstruct%grid_type<4 .and. .not. gridstruct%bounded_domain) then
! Prevent accumulation of rounding errors at overlapped domain edges:
       call mpp_get_boundary(u, v, domain, ebuffery=ebuffer,  &
                             nbufferx=nbuffer, gridtype=DGRID_NE )
`
So, any clarification and explanation are appreciated.

**Comment by ShawnCebulaNOAA** [comment]
Hello @TingLei-NOAA!
Were you able to get the answer to your question?  If not, I can see if I can some answers for you!

Shawn Cebula - EPIC End User Support

**Reply by TingLei-NOAA** [reply]
@ShawnCebulaNOAA Yes. That will be great if some expert can give an answer on that.  I am still trying to learn more about fms. 
Thank you so much for your attention on this question.

**Reply by RatkoVasic-NOAA** [reply]
@TingLei-NOAA is this problem solved? It was long time ago.

**Comment by TingLei-NOAA** [comment]
@RatkoVasic-NOAA Sure. The Jeid 's D-A wind conversions issue had been fixed by D. Holdaway with the help from R. Benson. I think this question could be closed.  Thanks
